tool/cli.js

```diff
@@ -106,18 +106,8 @@ program
   )
 
   .command("add-redirect", "Add a new redirect")
-  .argument("<from>", "From-URL", {
-    validator: (value) => {
-      Redirect.validateFromURL(value, false);
-      return value;
-    },
-  })
-  .argument("<to>", "To-URL", {
-    validator: (value) => {
-      Redirect.validateToURL(value, false);
-      return value;
-    },
-  })
+  .argument("<from>", "From-URL")
```

Comment:
We validate first thing when calling `Redirect.add` and we would have to parse the locale out of the URL otherwise. In general this breaks easy so let's just use the validation in `Redirect.add`.